typings/index.d.ts

```diff
@@ -146,6 +146,9 @@ export interface Actions {
   ) => void
   clearSelection: (cb?: Function) => void
   clearItems: () => void
+  setItemCount: (count: number) => void
+  unsetItemCount: () => void
+  setState: (stateToSet: object, cb?: Function) => void
```

Comment:
isn't setState defined by React.Component?

Reply:
In a recent change we exposed our `internalSetState` method as an action (called it `setState`).